code/controllers/subsystem/mapping.dm

```diff
@@ -82,6 +82,7 @@ SUBSYSTEM_DEF(mapping)
 			config = old_config
 	initialize_biomes()
 	loadWorld()
+	generate_z_level_linkages()
```

Comment:
you should move this into InitializeDefaultLevels
loadWorld news a lot of things, including potentially initializing stuff that will attempt to use the list

Reply:
Done.